com.milesalan.mepo.yml

```diff
@@ -0,0 +1,110 @@
+# Flatpak Manifest
+app-id: com.milesalan.mepo
+runtime: org.freedesktop.Platform
+runtime-version: '21.08'
+sdk: org.freedesktop.Sdk
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.ziglang
+rename-icon: mepo
+rename-desktop-file: mepo.desktop
+finish-args:
+  - --device=dri
+  - --share=ipc
+  - --socket=x11
+  - --socket=wayland
+  - --share=network
+  - --filesystem=xdg-documents
+
+build-options:
+  append-path: /usr/lib/sdk/ziglang
+command: mepo
+modules:
+  # Imagemagick for icon size conversion
+  - name: imagemagick
+    cleanup: '*'
+    sources:
+      - type: archive
+        url: https://github.com/ImageMagick/ImageMagick/archive/refs/tags/7.1.0-47.tar.gz
+        sha256: ad489af583bdfc0b5edd6c2106c955bf3af68fe838f6bd32c0ee8505ec6891ff
+
+  # Core build deps for mepo (note SDL2, SDL2_image, and SDL2_ttf is in runtime)
+  - name: SDL2_gfx
+    buildsystem: autotools
+    sources:
+      - type: archive
+        url: http://www.ferzkopp.net/Software/SDL2_gfx/SDL2_gfx-1.0.4.tar.gz
+        sha256: 63e0e01addedc9df2f85b93a248f06e8a04affa014a835c2ea34bfe34e576262
+      - type: shell
+        commands:
+          - cp -p /usr/share/automake-*/config.{sub,guess} .
+          - autoreconf -fvi
+          - ./configure --prefix=/usr --disable-mmx
```

Comment:
`/usr` isn't the prefix

```suggestion
```

Reply:
Changed to `--prefix=/app`, is that right?

Reply:
Oh I see other comment so this doesn't apply